futures-util/src/io/window.rs

```diff
@@ -65,43 +65,36 @@ impl<T: AsRef<[u8]>> Window<T> {
         self.range.end
     }
 
-    /// Changes the starting index of this window to the index specified.
+    /// Changes the range of this window to the range specified.
     ///
     /// Returns the windows back to chain multiple calls to this method.
```

Comment:
not new to this change, but why would you want to chain multiple calls to this method?

Reply:
That's certainly not necessary because we can now set the range at once with `set`.